proseco/tests/test_guide.py

```diff
@@ -72,7 +72,8 @@ def test_box_mag_spoiler():
     assert 688523960 in selected1['id']
 
     # Set the spoiler to be 10th mag and closer to the second star
-    stars.get_id(688522000)['mag'] = 10.0
+    stars.get_id(688522000)['mag'] = 9
+    stars.get_id(688522000)['row'] = 50
```

Comment:
Why these changes?  I thought all updates were strictly about adding debug info.

Reply:
As far as I could tell, this test was no longer actually testing what it said it was.  Didn't notice until more debug info.  The original comment mentioned moving the star closer which seems to have gotten cut at some point, and the mag difference wasn't actually enough to hit the box test either.